sphinx-docs/usermanual/userman_admin.rst

```diff
@@ -0,0 +1,331 @@
+
+ 
+.. WARNING:: 
+    If you chose to download them to somewhere other than the content folder inside the ka-lite folder as recommended above, you need to tell KA Lite where to find them. If this is the case, follow the steps below:
+
+
+#. Make sure all video files are located in a single directory, with .mp4 extensions (KA Lite expects this!)
+#. If it doesn't already exist, create a file named local_settings.py in the ka-lite/kalite folder (the one containing settings.py)
+#. Add the line ``CONTENT_ROOT="[full path to your videos directory]"``, making SURE to include an OS-specific slash at the end (see examples) and encapsulate it in quotes. 
+    **For example, on Windows:** ``CONTENT_ROOT="C:\\torrented_videos_location\\"``
+
+    **For example, on Linux:** ``CONTENT_ROOT="/home/me/torrented_videos_location/"``
+#. Restart your server. If you are unsure on how to do this, please see `Restarting Your Server`_ .
+
+.. _BitTorrent Sync: http://www.getsync.com/
+
+
+Language Packs
+---------------------------
+KA Lite supports internationalization. You can download language packs for your language if it is available. A language pack comes with all the available subtitles and user interface translations for that language. When it is installed, KA Lite will give you the option to download individual dubbed videos from the language's Khan Academy YouTube channel.
+
+Download Language Packs
+^^^^^^^^^^^^^^^^^^^^^^^^^
+To download language packs: 
+
+#. From the "Manage" page, click on the "Language" tab. 
+#. Select the language pack you wish to download by selecting from the drop-down menu.
+#. Click the "Get Language Pack" button. 
+#. Once the download finishes, you can see your language pack in the list of installed packs. Learners and coaches will now be able to switch their language to any of the installed language packs. Their default will be the default that you set by clicking on "Set as default". 
+
+Delete Language Packs
+^^^^^^^^^^^^^^^^^^^^^^^^^
+To delete language packs:
+
+#. Log in as the administrator.
+#. Click the "Languages" link in the navigation bar
+#. In the Installed Languages section, there is a button for deletion of each language.
+
+
+Restarting Your Server
+-----------------------
+If you have made some configuration changes (such as changing the filepath to your video content to your liking), or if you feel the need to reboot your KA Lite system, you may want to restart your server. Please note that this will cause KA Lite to become inaccessible to any users. However, this will not delete any user accounts or information that you have configured during set up. 
+
+This process varies, depending on which OS you are running the KA Lite Server on. 
+
+Restarting Your Server: Windows
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+Restarting Your Server: Linux
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+Restarting Your Server: Mac 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
@tk21 can you try out the `bin/kalite` executable and see if it's working in mac? Then we can update this part of the manual.